src/ClientHandle.cpp

```diff
@@ -1924,7 +1924,7 @@ void cClientHandle::Tick(float a_Dt)
 		{
 			m_PingID++;
 			m_PingStartTime = std::chrono::steady_clock::now();
-			m_Protocol->SendKeepAlive(m_PingID);
+			m_Protocol->SendKeepAlive(static_cast<UInt32>(m_PingID));
```

Comment:
When varints were first implemented in Cuberite, the documentation only supported unsigned varints, so that's what was implemented. Signed ones were documented much later and so far are not implemented.

Since the actual value of the ping ID is not relevant, only that it is the same as the one sent, I suggest we convert it to `UInt32`.